doc/api_assets/style.css

```diff
@@ -1,5 +1,6 @@
 /*--------------------- Layout and Typography ----------------------------*/
 html {
+  -webkit-overflow-scrolling: touch;
```

Comment:
I'm not familar with the way IOS scrolls, but do I understand correctly that momentum scrolling is disabled by default on web pages and enabled in apps?